docs/docs/reference/dropped-features/package-objects.md

```diff
@@ -40,3 +40,20 @@ If a source file `src.scala` contains such toplevel definitions, they will be pu
 **Note 2:** A toplevel main method `def main(args: Array[String]): Unit = ...` is wrapped as any other method. If it appears
 in a source file `src.scala`, it could be invoked from the command line using a command like `scala src$package`. Since the
 "program name" is mangled it is recommended to always put `main` methods in explicitly named objects.
+
+### Private toplevel definitions
+
+If a private toplevel definition is wrapped in a synthetic object, it is visible only
+in definitions in the same source file. By contrast, a package-private definition is visible in the whole containing package. Example:
```

Comment:
Is a `private` definition also visible inside `object`s, `class`es and `trait`s that are in the same source file? It should, but the wrapping inside the source-package-object could jeopardize that. Worth a test?

Reply:
Ouch. That actually ruins the scheme I am afraid.